apis/runtime/v1alpha1/api.proto

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// To regenerate api.pb.go run scripts/generate_runtime.sh
+syntax = "proto3";
+
+package runtime.v1alpha1;
+option go_package = "github.com/koordinator-sh/koordinator/apis/runtime/v1alpha1";
+
+
+// PodSandboxMetadata holds all necessary information for sandbox name.
+message PodSandboxMetadata {
+  // Pod name of the sandbox. Same as the pod name in the Pod ObjectMeta.
+  string name = 1;
+  // Pod UID of the sandbox. Same as the pod UID in the Pod ObjectMeta.
+  string uid = 2;
+  // Pod namespace of the sandbox. Same as the pod namespace in the Pod ObjectMeta.
+  string namespace = 3;
+  // Attempt number of creating the sandbox. Default: 0.
+  uint32 attempt = 4;
```

Comment:
remove this filed

Reply:
reverve for sequence features, such as ingoring core adjustment when attempt > 1

Reply:
> remove this filed

reverve for sequence features, such as ingoring core adjustment when attempt > 1